src/ElevatorButton.java

```diff
@@ -0,0 +1,16 @@
+public class ElevatorButton {
+  FloorLamp lamp;
+
+  public ElevatorButton() {
+    lamp = new FloorLamp(false);
+  }
+
+  public void click() {
+    if(lamp.isOn()) {
+      lamp.setOn(false);
```

Comment:
you can simplify the logic in this method by doing 
```suggestion
      lamp.setOn(!lamp.isOn());
```